packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -306,14 +313,17 @@ export class VaTableInner {
   }
 
   render() {
-    const { tableTitle, tableType, stacked } = this;
+    const { tableTitle, tableType, stacked, scrollable } = this;
     const classes = classnames({
       'usa-table': true,
       'usa-table--stacked': stacked,
       'usa-table--borderless': tableType === 'borderless',
     });
     return (
-      <div>
+      <div
+        tabIndex={scrollable ? 0 : undefined}
+        class={scrollable ? 'usa-table-container--scrollable' : undefined}
```

Comment:
Instead of `undefined`, can we instead us `null`?  Null would represent the intentional absence of a value which I think better describes how we want to assign a value to these attributes.